BUILD.md

```diff
@@ -6,22 +6,23 @@ instructions for building it.
 ## Atomic.app
 
 The recommended way to test the site is with [atomic.app](https://github.com/projectatomic/atomicapp).  Currently,
-verion 0.4.2 or later is required.
+version 0.4.2 or later is required.
 
 ### First time setup
 
-There are a few setup steps you need to do the first time you clone
-the repository and run this as an atomic app.  You should have to do these again
-unless you re-clone the repo for some reason.
+The first time you run it in Atomic App, you'll need to generate an answers.conf
+file with information it needs to launch the container.  You shouldn't have to
+do this on successive runs unless you delete the answers.conf file for some
+reason.
 
-1. authorize directory sharing in SELinux (may not be required in a VM):
+1. run genanswers:
 
-```
-    ./selinux_share.sh
-```
+   ```
+   cd /path/to/atomic-site/Nulecule/
+   atomicapp genanswers .
```

Comment:
run immediately into this: https://github.com/projectatomic/atomicapp/issues/652